x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/utility_bar.test.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { ThemeProvider } from 'styled-components';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import { waitFor } from '@testing-library/react';
+
+import { AllRulesUtilityBar } from './utility_bar';
+
+describe('AllRules', () => {
+  it('renders AllRulesUtilityBar total rules and selected rules', () => {
+    const wrapper = mount(
+      <ThemeProvider theme={() => ({ eui: euiDarkVars, darkMode: true })}>
```

Comment:
nit: In tests I think the pattern of declaring a theme at the top:

```ts
const theme = () => ({ eui: euiDarkVars, darkMode: true });
```

And then using it everywhere is better:

```ts
<ThemeProvider theme={theme}>
```

It's just less syntax noise and repetition. At least that's the pattern I follow at the moment.

Reply:
👍  thanks, that's a nice pattern. Updated in https://github.com/elastic/kibana/pull/83023